src/models/tasks.py

```diff
@@ -31,7 +31,7 @@ def __init__(
             num_attention_heads=config.n_head,
             hidden_dropout_prob=config.dropout,
             attention_probs_dropout_prob=config.dropout,
-            num_classes=config.num_classes,
+            num_labels=config.num_classes,
```

Comment:
Are you sure about this change?

Reply:
yes, 100% - there is no num_classes attribute in Bert configuration
(and we did these changes in master already)